e2e/vmss.go

```diff
@@ -115,45 +116,56 @@ func extractLogsFromVMLinux(ctx context.Context, s *Scenario) {
 	require.NoError(s.T, err)
 
 	commandList := map[string]string{
-		"cluster-provision":            "cat /var/log/azure/cluster-provision.log",
-		"kubelet":                      "journalctl -u kubelet",
-		"cluster-provision-cse-output": "cat /var/log/azure/cluster-provision-cse-output.log",
-		"sysctl-out":                   "sysctl -a",
-		"aks-node-controller":          "cat /var/log/azure/aks-node-controller.log",
+		"cluster-provision.log":            "cat /var/log/azure/cluster-provision.log",
+		"kubelet.log":                      "journalctl -u kubelet",
+		"cluster-provision-cse-output.log": "cat /var/log/azure/cluster-provision-cse-output.log",
+		"sysctl-out.log":                   "sysctl -a",
+		"aks-node-controller.log":          "cat /var/log/azure/aks-node-controller.log",
 	}
 
-	podName, err := getHostNetworkDebugPodName(ctx, s.Runtime.Cluster.Kube, s.T)
+	pod, err := s.Runtime.Cluster.Kube.GetHostNetworkDebugPod(ctx, s.T)
 	if err != nil {
 		require.NoError(s.T, err)
 	}
 
 	var logFiles = map[string]string{}
 	for file, sourceCmd := range commandList {
-		s.T.Logf("executing command on remote VM at %s: %q", privateIP, sourceCmd)
-
-		execResult, err := execOnVM(ctx, s.Runtime.Cluster.Kube, privateIP, podName, string(s.Runtime.SSHKeyPrivate), sourceCmd)
+		execResult, err := execOnVM(ctx, s.Runtime.Cluster.Kube, privateIP, pod.Name, string(s.Runtime.SSHKeyPrivate), sourceCmd)
 		if err != nil {
-			s.T.Logf("error fetching logs for %s: %s", file, err)
+			s.T.Logf("error executing %s: %s", sourceCmd, err)
 			continue
 		}
 		if execResult.stdout != nil {
 			out := execResult.stdout.String()
 			if out != "" {
-				logFiles[file+".stdout.txt"] = out
+				logFiles[file] = out
 			}
 
 		}
 		if execResult.stderr != nil {
-			out := execResult.stderr.String()
+			out := removeWarningMessages(execResult.stderr.String())
 			if out != "" {
-				logFiles[file+".stderr.txt"] = out
```

Comment:
why are we now putting the stdout and stderr in teh same log file ?   Should we add a delimited at the beginning and end of each log type ?

```
=================
           STDOUT
=================
```

Reply:
It's not the same file.

Stdout is saved into a file. Stderr (if any) is going to a log stdout. (Maybe combining stdout a stderr saving in a single file is better)

99% of the time the stderr from execution contains:

```
Warning: Permanently added '10.224.0.6' (ED25519) to the list of known hosts.

Authorized uses only. All activity may be monitored and reported.
```

It's just a noise to sift through that I wanted to get rid of.

Reply:
changed to write combined output to the same file. (So, I don't need to see noise in test logs, and don't need to download 2 files from CI)